staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -68,6 +68,7 @@ func newService(name string, serviceType v1.ServiceType, tweaks ...serviceTweak)
 		},
 		Spec: v1.ServiceSpec{
 			Type: serviceType,
```

Comment:
```suggestion
			Type:  serviceType,
```

You can run `./hack/update-gofmt.sh` as well.

Reply:
ok, thx~